src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html

```diff
@@ -207,7 +207,7 @@ <h3 class="inline">Results</h3>
           </div>
         </div>
         <div *ngIf="element.knockouts.length >= 10">
-          <a (click)="showAllKnockouts = true;" [hidden]="showAllKnockouts" style="cursor: pointer; padding-top: 10px">
+          <a (click)="showAllKnockouts = true;" [hidden]="showAllKnockouts">
```

Comment:
did you create a class for the style?

Reply:
I added style for all <a> elements, and now there is a pointer also when user hovers on the length of the reactions/genes to expand the row with the detailed info